File: src/LinksNews/Services/Data/Models/Account.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace LinksNews.Services.Data.Models
{
    [Table("account")]
    public class Account
    {
        public long? Id { get; set; }
        public long RoleId { get; set; }
        public string Login { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public long? LanguageId { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public string Telephone { get; set; }
        public string Website { get; set; }
        public string Comment { get; set; }
        public long? ThemeId { get; set; }
        public bool Locked { get; set; }
        public DateTimeOffset DateCreated { get; set; }
        public long? NewsRefreshInterval { get; set; }

        [NotMapped]
        public string Password { get; set; }
        [NotMapped]
        public string ImageUrl { get; set; }

        public string ToName
        {
            get
            {
                string result = null;

                if (!string.IsNullOrEmpty(FirstName))
                {
                    result = FirstName.Trim();
                    if (!string.IsNullOrEmpty(LastName))
                    {
                        result += " " + LastName.Trim();
                    }
                }

                return result ?? Login;
            }
        }

    }
}

File: src/LinksNews/angular2App/app/core/domain/account.ts
import { Language } from "./language";
import { Theme } from "./theme";
import * as _ from 'lodash';

export class Account {
    id: number;
    roleId: number;
    login: string;
    //password: string;
    firstName: string;
    lastName: string;
    languageId: number;
    email: string;
    address: string;
    telephone: string;
    website: string;
    comment: string;
    themeId: number;
    locked: boolean;
    dateCreated: Date;
    imageUrl: string;
    newsRefreshInterval: number;

    clone(): Account {
        let result: Account = <Account>_.assign(new Account(), this);
        return result;
    }

    static from (account: Account): Account {
        
        let result: Account = new Account();
        _.assign(result, account);
        return result;
    }
}
